### PR TITLE
New supplier locations

### PR DESCRIPTION
**Pull request by martyn-w**
New locations <--> supplier mappings
based on the latest spreadsheets from the programme

### CODE REVIEW

File: lib/tasks/data/supplier_locations.yml [hunk 1]
```diff
@@ -229,7 +399,7 @@ suppliers:
     - CAM4
     - CAM5
     - CDI
-    - CITWMC
```
Comment (martyn-w) [inline lib/tasks/data/supplier_locations.yml:232]: `CITWMC` is Westminster Mag Court and should not have been in this list as its a court

File: lib/tasks/data/supplier_locations.yml [hunk 2]
```diff
@@ -243,51 +413,99 @@ suppliers:
...
+    - GCS2
+    - GCS3
+    - GCS4
     - GCS5
+    - GCS6
     - GMI
     - GNI
     - HBI
     - HCI
+    -
```
Comment (smoothcontract) [inline lib/tasks/data/supplier_locations.yml:466]: Not sure if having this blank line will cause an issue? Probably best to delete just to be safe.

**Review by smoothcontract — COMMENTED** [review]
Looks good - loads of new locations! Just one extra line which I think it would be safest to remove.

File: lib/tasks/data/supplier_locations.yml (hunk 2 above)
Comment (martyn-w) [reply lib/tasks/data/supplier_locations.yml:466]: whoops, that should not be there, good spot

**Review by smoothcontract — APPROVED** [review]
LGTM 👍

**Pull request merged**